spec/unit/shoulda/matchers/active_model/validate_presence_of_matcher_spec.rb

```diff
@@ -327,7 +327,7 @@ def model_creator
     end
   end
 
-  if rails_4_x?
+  if rails_4_x? || rails_5_x?
```

Comment:
Good thinking. In order to merge this, we will also need tests for the `#{attribute}_digest` case. Unfortunately, in order for you to even run those tests, we will need to add Rails 6 to the Appraisals file. We can't do this quite yet until we have 4.0 released, so we will have to come back around to this.